k2/csrc/array_ops.h

```diff
@@ -295,8 +295,10 @@ void Or(Array1<T> &src, T default_value, Array1<T> *dest) {
 
 /*
   Returns a random Array1, uniformly distributed betwen `min_value` and
-  `max_value`.  CAUTION: this will be randomly generated on the CPU, for now,
-  and transferred to the CPU, so it will be slow if c is not a CPU context.
+  `max_value`.  CAUTION: for now, this will be randomly generated on CPU and
+  then transferred to other devices if c is not a CPU context, so it will be
+  slow if c is not a CPU context.
+  Note T should be floating-pointer type or integral type.
```

Comment:
This can be enforced at compile time with `static_assert` and `type_trais`.

Reply:
added